analytics_dashboard/static/sass/_developer.scss

```
@@ -600,7 +600,7 @@ table.dataTable thead th.sorting_desc:after {
   }
 
   a.disabled {
-    @extend disabled-link;
+    @extend disabled-link !optional;
```

Can anyone ( @dsjen ) can give insight into where this rule is used? I can't find anything with the class 'chart-marginals' in Insights. I think one of the latest `libsass` versions started to throw errors where it used to silently fail and this is one of those cases. Adding `!optional` silences the error again, but I'm afraid I might have broke whatever style this is for.